forte/data/ontology/configs/example_ontology_config.json

```diff
@@ -1,25 +1,26 @@
 {
-    "ontology_name": "example_ontology",
-    "imports": [
-       "example_import_ontology_config.json"
+    "name": "example_ontology",
+    "description": "",
+    "import_paths": [
+        "example_import_ontology_config.json"
     ],
-    "entry_definitions": [
+    "definitions": [
         {
-            "entry_name": "forte.data.ontology.example_ontology.Token",
-            "parent_entry": "forte.data.ontology.example_import_ontology.Token",
-            "entry_description": "",
+            "entry_name": "example_ontology.Word",
+            "parent_type": "example_import_ontology.Token",
+            "description": "",
             "attributes": [
                 {
-                    "attribute_name": "related_tokens",
-                    "attribute_description": "Tokens related to the current token",
-                    "attribute_type": "List",
-                    "element_type": "forte.data.ontology.example_ontology.Token"
+                    "name": "string_features",
+                    "description": "To demonstrate the composite type, List.",
+                    "type": "List",
+                    "item_type": "str"
                 },
                 {
-                    "attribute_name": "string_features",
-                    "attribute_description": "Miscellaneous string features",
-                    "attribute_type": "List",
-                    "element_type": "str"
+                    "name": "word_forms",
+                    "description": "To demonstrate that an attribute can have the entry it is contained in as it's type or item_type.",
+                    "type": "List",
+                    "item_type": "example_ontology.Word"
```

Comment:
The test ontology is not quite sufficient. Could you try adding a couple more types? We could test the following:

1. Multiple entry definitions, this only contains one.
1. Entry definitions inherited from types form the same file (e.g Try extend word to something else?)
1. Try to use each type at least once  (i.e. str, int, bool, float, List)

You may be able to think of other points here. The point is, we would like to have a fairly comprehensive test, without assuming they would all work.